Documentation/Blazorise.Docs/Pages/News/2024-10-15-release-notes-170.razor

```diff
@@ -312,4 +312,12 @@
     We have improved the way how our Fluent Flex utilities work. Previously it was not possible to define a breakpoint on per-flex rule, which made it hard to define different flex rules for different breakpoints. Now you can define the flex rules for each breakpoint separately, and you can also define the order of the flex rules.
 </Paragraph>
 
+<Heading Size="HeadingSize.Is3">
+    Disable Video Fullscreen
```

Comment:
I'd still add "Disable Video Fullscreen on Double Click". But I guess it's fine to keep the Heading simpler if you prefer.